tests/Microsoft.Graph.DotnetCore.Core.Test/Microsoft.Graph.DotnetCore.Core.Test.csproj

```diff
@@ -46,7 +46,7 @@
 		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
 		<PackageReference Include="Moq" Version="4.10.1" />
 		<PackageReference Include="xunit" Version="2.4.1" />
-		<PackageReference Include="Microsoft.Graph" Version="1.*" />
+		<PackageReference Include="Microsoft.Graph" Version="3.14.0" />
```

Comment:
shouldn't this be 3.* instead of locking down the version?

Reply:
Yes. Will change.